python/torch_mlir_e2e_test/test_suite/quantize.py

```diff
@@ -0,0 +1,41 @@
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Also available under a BSD-style license. See LICENSE.
+
+import torch
+
+from torch_mlir_e2e_test.framework import TestUtils
+from torch_mlir_e2e_test.registry import register_test_case
+from torch_mlir_e2e_test.annotations import annotate_args, export
+
+# ==============================================================================
+
+
+class Quantize(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1], torch.float32, True),
+    ])
+    def forward(self, x):
+        y1 = torch.int_repr(torch.quantize_per_tensor(x, 1.0, 0, torch.qint8))
+        # FIXME: When lowering to TOSA, the type gets converted to int8
```

Comment:
This is to be fixed before upstreaming?

Reply:
I'm unsure. I don't want to make this PR to big, and having qint8 work is already a nice improvement.
What do you think?